fuzztests/runFuzzTests.js

```diff
@@ -9,7 +9,7 @@ const fs = require("fs/promises");
 const { spawn } = require("child_process");
 
 const fuzzTestFileExtension = "fuzz.js";
-const fuzzTestNameRegex = /it.fuzz\("(.*)"/g;
+const fuzzTestNameRegex = /it.fuzz\(\s*"(.*)"/g;
```

Comment:
Ah, good point. Should we allow whitespace before the parenthesis as well?

Reply:
I've added it. Maybe we need some metatests to check if all tests have ran :laughing: